src/pages/progress_reports/SubmitProgressReport.tsx

```diff
@@ -67,6 +68,21 @@ export default function SubmitProgressReport(props: SubmitProgressReportProps) {
     return dataManagementIncident;
   }
 
+
+  const getCloseoutInfo = (formState: FormState): Closeout => {
+    const closeout: Closeout = {} as Closeout;
+    closeout.reasons = []
+    CLOSEOUT_KEYS.map((key) => {
```

Comment:
Keeping this as a map instead of a forEach since we made the choice to do the same on line 60.